src/components/icon/icon.stories.mdx

```diff
@@ -154,6 +154,32 @@ See <LinkTo kind='Documentation/Colors' name="page">Colors</LinkTo> for more inf
   </Story>
 </Preview>
 
+### Without hover state
+
+<Preview>
+  <Story
+    name="without hover state"
+    parameters={{ info: { disable: true }, chromatic: { disable: true } }}
```

Comment:
You don't need either of these parameters on this story because they're both on the `<Meta />` tag at the top - that sets the parameters for every story.

Looks like some of the other stories in this file mistakenly have these as well!

Reply:
Good point, changed, thanks!